cea/demand/ventilation_model.py

```diff
@@ -409,7 +409,7 @@ def allocate_default_leakage_paths(coeff_lea_zone, area_facade_zone, area_roof_z
 
     coeff_lea_path = np.zeros(5)
     height_lea_path = np.zeros(5)
-    orientation_lea_path = np.zeros(5)
+    orientation_lea_path = [0, 1, 0, 1, 2]
```

Comment:
No. Right now it does not have any implications. As windows are not used anyways in any calculation. As far as I understand.
Why would you change that?
And what does this have to do with cooling coils?